securedrop/journalist_templates/col.html

```diff
@@ -31,28 +31,21 @@
 
       <ul id="submissions" class="plain submissions">
         {% for doc in source.collection %}
-          <li class="submission {% if not doc.downloaded %}unread{% endif %}">
-            {% if not doc.filename.endswith('reply.gpg') %}
-              {% if not doc.downloaded %}
+          <li class="submission {% if not doc.downloaded and not doc.filename.endswith('reply.gpg') %}unread{% endif %}">
+              {% if doc.filename.endswith('reply.gpg') %}
+                <input type="checkbox" name="doc_names_selected" value="{{ doc.filename }}" class="doc-check">
+                <span title="Reply" class="icon"><i class="fa fa-reply pull right"></i></span>
+              {% elif not doc.downloaded %}
                 <input type="checkbox" name="doc_names_selected" value="{{ doc.filename }}" class="doc-check unread-cb">
                 <span title="Unread" class="icon"><i class="fa fa-envelope"></i></span>
               {% else %}
-                <input type="checkbox" name="doc_names_selected" value="{{ doc.filename }}" class="doc-check">
-                <span class="icon"><i class="fa fa-envelope-open"></i></span>
+                <input type="checkbox" name="doc_names_selected" value="{{ doc.filename }}" class="doc-check unread-cb">
+                <span title="Read" class="icon"><i class="fa fa-envelope-open"></i></span>
               {% endif %}
-            {% elif doc.filename.endswith('reply.gpg') %}
-                <input type="checkbox" name="doc_names_selected" value="{{ doc.filename }}" class="doc-check">
-                <span class="icon"></span>
-            {% endif %}
-            {% if doc.filename.endswith('reply.gpg') %}
-              <span class="file reply"><span class="filename">{{ doc.filename }}</span></span>
-              <span class="info"><span title="{{ doc.size }} bytes">{{ doc.size|filesizeformat() }}</span></span>
-            {% else %}
-              <a class="file {% if not doc.downloaded %}unread{% else %}read{% endif %}" href="{{ url_for('col.download_single_submission', filesystem_id=filesystem_id, fn=doc.filename) }}">
-                <i class="fa fa-download"></i> <span class="filename">{{ doc.filename }}</span>
-              </a>
-              <span class="info"><span title="{{ doc.size }} bytes">{{ doc.size|filesizeformat() }}</span></span>
-            {% endif %}
+            <a class="file {% if not doc.downloaded and not doc.filename.endswith('reply.gpg') %}unread{% else %}read{% endif %}" href="{{ url_for('col.download_single_submission', filesystem_id=filesystem_id, fn=doc.filename) }}">
```

Comment:
Since we're using `col.download_single_submission` here, we'll need to modify the logic in that view function a bit due to the following `try`, `except` block that's in there now: 

```
        try:
            Submission.query.filter(
                Submission.filename == fn).one().downloaded = True
            db.session.commit()
        except NoResultFound as e:
            current_app.logger.error(
                "Could not mark " + fn + " as downloaded: %s" % (e,))
```

The issue here is that it's expecting a `Submission` object so that it can mark the file as downloaded. We only want to enter this `try`, `except` block if we are indeed downloading a `Submission` (not a `Reply`), otherwise we won't find the object, and we'll log an error (which will produce an OSSEC alert).

Also, since we're now using this `download_single_submission` logic to download both submissions and replies, I think for clarity we should rename it `download_single_file` or similar. What do you think?